src/componnents/GoodsList/GoodsList.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button';
+import Select from '../Select/Select';
+import List from '../List/List';
+
+const selectItems = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+class GoodsList extends Component {
+  state = {
+    goods: [...this.props.goods],
```

Comment:
```suggestion
    goods: this.props.goods,
```
тут не нужен новый массив